### PR TITLE
Added samples for the EditorModel events

### PR DESCRIPTION
**Pull request by abjerner**
Mostly for the content part for now.

I've taken the liberty to upload the images to GitHub as user content. Is this fine, or should I add them to this repo instead? I don't think GitHub ever deletes these files - but I could be wrong.

### CODE REVIEW

File: Reference/Events/EditorModel-Events/Customizing-tabs.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        {
+            tab.Label = "Hello World";
+        }
+
+    }
+
+};
+```
+
+In this Umbraco, we iterate over the tabs provided in the model, and change the label of the **Content** tab to **Hello World**.
```
Comment (jmayntzhusen) [inline Reference/Events/EditorModel-Events/Customizing-tabs.md]:
```suggestion
In Umbraco, we iterate over the tabs provided in the model, and change the label of the **Content** tab to **Hello World**.
```

File: Reference/Events/EditorModel-Events/Customizing-tabs.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+For another scenario, we could have an **Advanced*' tab we only wish to show to for certain users:
```
Comment (jmayntzhusen) [inline Reference/Events/EditorModel-Events/Customizing-tabs.md]:
```suggestion
For another scenario, we could have an **Advanced** tab we only wish to show to for certain users:
```

File: Reference/Events/EditorModel-Events/Customizing-tabs.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        e.Model.Tabs = e.Model.Tabs.Where(x => x.Label != "Advanced");
+    }
+
+};
+```
+
+Notice that the example doesn't check the content type, so will apply for all **Advanced** tabs across all content items where a tab with this name is present.
+
+:::tip
+The `CssClass` property as shown in the example below is first available from Umbraco 7.13.
```
Comment (jmayntzhusen) [inline Reference/Events/EditorModel-Events/Customizing-tabs.md]:
```suggestion
The `CssClass` property as shown in the example below is available from Umbraco 7.13.
```

**Comment by jmayntzhusen** [comment]
Hey @abjerner 

We discussed the Github image thing at our Curators meeting yesterday, we would prefer if you keep uploading the images like normally - this gives the added benefit of the documentation working without an internet connection and also keeps at all bundled together nicely 😉

**Comment by dampee** [comment]
Hi @abjerner @jmayntzhusen 

I've moved the image to the repo.  Also added a "pre 7.4" document.  But could not come up with a better solution.  (please review).
I also cleaned up the markdown a little bit.  :-)

File: Reference/Events/EditorModel-Events/Customizing-tabs.md (hunk 2 above)
Comment (abjerner) [reply Reference/Events/EditorModel-Events/Customizing-tabs.md]: My bad. I should have seen this 😊

File: Reference/Events/EditorModel-Events/Customizing-tabs.md (hunk 1 above)
Comment (abjerner) [reply Reference/Events/EditorModel-Events/Customizing-tabs.md]: I guess my mind was full of Umbraco. It should have been "In this example". Should be fixed now ;)

**Comment by abjerner** [comment]
@dampee @jmayntzhusen thanks for the input. I should now have fixed in remaining issues 😉

**Comment by dampee** [comment]
Merged 👍 

Thanks a lot @abjerner 

This is very helpful information for a lot of people!

**Comment by abjerner** [comment]
@dampee Thanks 😄 

By the way, in addition to the samples, would it be an idea to link to external content as well?

For instance, Paul as an article about using the `SendingContentModel` event so set default values, which might be useful as well:

https://codeshare.co.uk/blog/how-to-add-default-values-to-umbraco-properties-in-the-backoffice/

image

Either by mentioning his name, or as it's external content, mention the domain instead.

I think there was a discussion at some point about having a **Read more** section, which could be used for external content. I haven't been able to find this discussion (don't remember whether it happened online or IRL), and so far I've only been able to find these two pages with a similar **Read more** section:

https://github.com/umbraco/UmbracoDocs/blob/master/Umbraco-Cloud/Set-Up/Manage-Hostnames/Umbraco-Latch.md

https://github.com/umbraco/UmbracoDocs/blob/master/Umbraco-Cloud/Set-Up/Manage-Hostnames/Security-Certificates/index.md

Only one of them links to external content, and that external content is just on *umbraco.com*.

Any thoughts?

**Comment by jmayntzhusen** [comment]
You may be thinking of this issue #1072 - I think attributing them by the author name would be the best, but I think this is something we should bring up the next time the Docs Curators meet up (next week Thursday 🙂)

**Comment by abjerner** [comment]
@jmayntzhusen it wasn't that one - but that sounds amazing. I'll await the result 😉

**Comment by dawoe** [comment]
@abjerner 

The links to the samples on this page are broken : https://our.umbraco.com/Documentation/Reference/Events/EditorModel-Events/Index/

Want to reference them in my dk meetup talk next week. Will also add some other examples when I find the time

**Comment by jmayntzhusen** [comment]
Hmm I think the broken links is due to the main page being named Index.md not index.md, this causes the URL to mess it up a bit and go to /Reference/Events/EditorModel-Events/Index/ not /Reference/Events/EditorModel-Events/

Will try to fix this 🙂

**Pull request merged**